### DISCUSSION TITLE
SX1276 Lower power receive - battery operated device

### DISCUSSION THREAD

**Discussion by SanderM2** [jgromes/RadioLib#826] [General]
I'm building a battery operated device that needs to be able to send/receive data to another (same) device. For this I'm currently planning on use the Arduino MKR WAN 1310 which is using the SX1276 LoRa chip.

Now, after reading the SX1276 datasheet it seems that the device is capable to run on it's internel RC Oscillator when waiting for a new packet. And considerably less power is used when it's running on it's RC Oscillator. But I don't think this is implemented in the Arduino LoRa library, so it will probably be using too much power for a battery operated device.

While looking in your code you seem to refer to the RC Oscillator which makes me think that using your library is probably going to consume much less power when the device is constantly listening for data because it'll be using it's RC Oscillator which is only a couple of uA of power.

My devices will be sending some data bytes at random moments so the other device will need to be listening for it all the time...

**Comment by jgromes** [comment]
I don't think the RC oscillator can be used for active reception, definitely not for LoRa (for FSK sequencer maybe) - where exactly did you get this information (e.g. page number in the datasheet)?

**Comment by HeadBoffin** [comment]
The MRK WAN 1310 is a SAMD for the main MCU with a Murata CMWX1ZZABZ module (STM32L07 + SX1276) acting as an AT cmd driven modem. There is a pass-through hack somewhere but overall, this may not be the board you are looking for - or can even drive with RadioLib. And getting below ~50uA sleep is a challenge so not great on batteries.
